### ISSUE TITLE
Gliden64 blue lines in perfect dark on 1964 emulator

### ISSUE THREAD

**Issue by cereal-person** [gonetz/GLideN64#2376]
specs: ryzen 5 3600 with xfx rx580. When I play perfect dark using the 1964 emulator so I can use mouse and keyboard, I get blue "gaps" or lines on walls or floors. It doesn't make the game unplayable but it is annoying. This could be because my setup isn't the most powerful

**Comment by carnivoroussociety** [comment]
Same issue? https://github.com/gonetz/GLideN64/issues/2248

**Comment by cereal-person** [comment]
> Same issue? #2248

Yes this is the same issue I’m having

**Comment by oddMLan** [comment]
Close the issue since it's a duplicate

**Comment by CadetSparklez** [comment]
Yeah the reason they are blue is that is the colour of perfect dark's nothing so it doesnt need to add in a skybox on certain levels. I forgot which it was but in a certain version of glide when you turned off a certain buffer or set it to color instead of vertical interrupt (forgot which one) when the game opens up the intro would be this blue colour, and once the menu opened it would be black instead.
